src/api/app/services/notification_creator.rb

```diff
@@ -5,23 +5,36 @@ class NotificationCreator
                       'Event::CommentForProject',
                       'Event::CommentForPackage',
                       'Event::CommentForRequest'].freeze
+  CHANNELS = [:web, :rss].freeze
```

Comment:
for email we don't create a notification but instead just send the email, right?

Reply:
yes, we just send the email